doc/source/reference/arrays.classes.rst

```diff
@@ -6,17 +6,10 @@ Standard array subclasses
 
 .. currentmodule:: numpy
 
-The :class:`ndarray` in NumPy is a "new-style" Python
-built-in-type. Therefore, it can be inherited from (in Python or in C)
-if desired. Therefore, it can form a foundation for many useful
-classes. Often whether to sub-class the array object or to simply use
-the core array component as an internal part of a new class is a
-difficult decision, and can be simply a matter of choice. NumPy has
-several tools for simplifying how your new object interacts with other
-array objects, and so the choice may not be significant in the
-end. One way to simplify the question is by asking yourself if the
-object you are interested in can be replaced as a single array or does
-it really require two or more arrays at its core.
+Subclassing a ``numpy.ndarray`` is possible but if your goal is to create an
```

Comment:
Agreed. Reworking this properly is a bigger job. I have restored the original introduction, with only minor edits, and placed a note at the top directing users to consider dispatch instead of subclassing.